wasm example fails to build

Using the current main branch, the wasm example fails to build. This also happens when I try to compile a wasm project with FutureSDR as a dependency. I will try to figure out what's the issue and report back. Any hints on where to look are welcome!

```
error[E0433]: failed to resolve: could not find `UsbControlTransferParameters` in `web_sys`
   --> /home/user/Git/FutureSDR_main/src/blocks/wasm/hack_rf.rs:249:34
    |
249 |         let parameter = web_sys::UsbControlTransferParameters::new(
    |                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ could not find `UsbControlTransferParameters` in `web_sys`
```

This indicates that web-sys is somehow not working, maybe a missing feature.

--- a/examples/wasm/.cargo/config.toml
+++ b/examples/wasm/.cargo/config.toml
@@ -1,0 +1,2 @@
+[build]
+rustflags = ["--cfg=web_sys_unstable_apis", "-C", "target-cpu=generic"]

--- a/examples/wasm/Makefile
+++ b/examples/wasm/Makefile
@@ -1,6 +1,6 @@
 .PHONY: build
 
 build:
-	RUSTFLAGS='-C target-cpu=generic' wasm-pack build --target web --out-name wasm --out-dir ./dist --dev
+	wasm-pack build --target web --out-name wasm --out-dir ./dist --dev
 	./gulp default serve
 

